src/blockdata/opcodes.rs

```diff
@@ -659,19 +659,20 @@ impl fmt::Debug for All {
     }
 }
 
-
-/// Classification context for the opcode. Some opcodes like `OP_RESERVED`
-/// abort the script in [`ClassifyContext::Legacy`] context, but will act as OP_SUCCESS in tapscript
+/// Classification context for the opcode.
+///
+/// Some opcodes like `OP_RESERVED` abort the script in [`ClassifyContext::Legacy`] context, but
+/// will act as `OP_SUCCESS` in tapscript.
```

Comment:
Nit:
```suggestion
/// Some opcodes like [`All::OP_RESERVED`] abort the script in [`ClassifyContext::Legacy`] context, but
/// will act as [`All::OP_SUCCESS`] in tapscript.
```

Reply:
This brought to my attention the fact that `OP_SUCCESS` does not exist. I was unable to work out what this comment should be. @sanket1729 `git  blame` points to you as author of this comment, do you remember what you meant when you wrote it please?

Reply:
In Tapscript all the opcodes that used to be OP_RETURN are now OP_SUCCESS. Arguably we should make renames for every one of them, similar to how `OP_PUSHDATA_0` and `OP_FALSE` are aliased. But this is not very convenient and it's hard to see how it would be useful.

I think just removing the link here and returning to `OP_SUCCESS` with backticks would be fine here. Actually we should rename it to `OP_SUCCESSx`; see https://github.com/bitcoin/bips/blob/master/bip-0342.mediawiki

Reply:
Done as suggested (along with a few other minor improvements to that rustdoc comment) in https://github.com/rust-bitcoin/rust-bitcoin/pull/877